ChangeLog.md

```diff
@@ -1,6 +1,32 @@
 
 # Change Log
 
+## Version 10.14.0
+
+### New features
+1. Feature to [permanently delete](https://docs.microsoft.com/en-us/rest/api/storageservices/delete-blob#remarks) soft-deleted
+   snapshots/versions of the blobs has been added (preview). `--permanent-delete=none/snapshots/version/snapshotsandversions`.
+2. Feature to preserve properties and ACLs when copying to Azure file share root directory.
+3. Pin all APIs to use the default service version `2020-04-08` and let users decide the service version via
```

Comment:
clarify that previously not all APIs respected the AZCOPY_DEFAULT_SERVICE_API_VERSION.